build-tasks/common.js

```diff
@@ -0,0 +1,141 @@
+const path = require('path');
```

Comment:
I have no idea for better file name. Any suggestion?

Reply:
How about splitting into `common/files.js` for file IO and `common/metadata.js` for everything else?